client/tests/integration/queries/role.rs

```diff
@@ -118,10 +120,17 @@ fn find_roles_by_account_id() -> Result<()> {
     let alice_id: <Account as Identifiable>::Id = "alice@wonderland".parse().expect("Valid");
 
     // Registering roles
+    println!("Registering roles");
```

Comment:
Perhaps format the roles themselves? This could verify the behaviour of the `Display` impl.

Reply:
Oops, it was a debug print. I'll remove it